provision.py

```diff
@@ -3,7 +3,7 @@
 import sys
 import logging
 import platform
-
+from install_phantomjs import install_phantomjs
```

Comment:
Probably simpler to just shell out to this as a script, since we're not passing in argument or anything.